kola/tests/etcd/discovery.go

```diff
@@ -113,7 +121,7 @@ func etcdMemberV2BackupRestore(c cluster.TestCluster) {
 
 	backup_to="$(mktemp -d)"
 
-	sudo etcdctl backup --data-dir=/var/lib/etcd \
+	sudo -E etcdctl backup --data-dir=/var/lib/etcd \
```

Comment:
Maybe limit it to just one env var?
```suggestion
	sudo --preserve-env=ETCD_ENABLE_V2 etcdctl backup --data-dir=/var/lib/etcd \
```

Reply:
cool, did not know that ! Thanks.